codestodians/food-basket-front-end
# Create custom components

#### What does this PR do?
- Creates custom components for the Food-Basket application.

#### Description of Task to be completed?
- Create `Text`, `Button`, and `Box` components

#### How should this be manually tested?
- Check out to the `feature-create-custom-components-153117957` branch.
- Click on either the `signin` or `signup` tabs to see the components rendered in the modal.
- Check the `CustomComponents` directory for created components.

#### Any background context you want to provide?
- Creating components that can be used generally through the app will save a lot of work going forward. Due to the eclectic props each of these components have, no matter the styling framework being used, we can still use the components. The passing of props becomes the only thing necessary for a component to be properly styled anywhere in the app.

- Also, we can gradually scale our components to the extent that at some time in the future, we rely solely on our own components. 

#### What are the relevant pivotal tracker stories?
- [#153117957](https://www.pivotaltracker.com/story/show/153117957)
